etna/generic_pages/tests/test_blocks.py

```diff
@@ -0,0 +1,54 @@
+import json
+import re
+
+from django.test import TestCase
+
+from wagtail.core.models import Site
+
+from ..models import GeneralPage
+
+
+class TestGeneral(TestCase):
+    def setUp(self):
+        root = Site.objects.get().root_page
+
+        self.general_page = GeneralPage(
+            title="General page",
+            body=json.dumps(
+                [
+                    {"type": "paragraph", "value": {"text": "This is a paragraph"}},
+                ]
+            ),
+        )
+        root.add_child(instance=self.general_page)
+
+    def assertContainsParagraph(self, subject: str, paragraph_text: str):
+        regex = re.compile(
+            "<div" + r"[\s\w'\=\-\"\.]*>\s*(.*)\s*<\/div" + ">",
+            re.MULTILINE,
+        )
+        paragraph_texts = []
+        for match in regex.finditer(subject):
+            paragraph_texts.append(match.groups()[0].strip())
+
+        self.assertIn(paragraph_text, paragraph_texts)
+
+    def test_view_url_accessible_by_name(self):
+        url = "/general-page/"
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
```

Comment:
What is it that we're testing here? It seems a little unclear.

Reply:
Hi Andy, so basically we can use the page name or the model name to do something like http://localhost:8000/general-page/

Reply:
But all this test is really doing is calling `self.client.get(url)` with a hardcoded value. The test after this one is doing the  exact same thing: calling `self.client.get(url)`... the only difference is that it gets the `"/general-page/"` string from the `get_url()` method beforehand.

Reply:
This is correct so maybe on the test_view_uses_correct_template , I can remove the   self.assertEqual(response.status_code, 200) , however I believe it is a good test for self.general_page.get_url()

Reply:
I would suggest removing this test completely, and just leaving the second one

Reply:
This test is now removed